fastlane/actions/apply_codestyle.rb

```diff
@@ -0,0 +1,54 @@
+require 'fileutils'
+
+module Fastlane
+  module Actions
+    class ApplyCodestyleAction < Action
+      def self.run(params)
+        UI.message("Format Objective-C code files in-place with clang-format tool")
+
+        begin
+          # download latest clang-format config
```

Comment:
this action is run when the code is changed to clang format, isn't it?

Reply:
yep, it's run from the format_objc lane. If the pre-commit hook fails, the error message printed includes `Run 'fastlane ios format_objc' to re-format or commit with --no-verify`

Reply:
yep, it's run from the format_objc lane. If the pre-commit hook fails, the error message printed includes `Run 'fastlane ios format_objc' to re-format or commit with --no-verify`